From b45e7cf776e455dfc7d99684c7c3efc1783adb32 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martinpitt@gnome.org>
Date: Thu, 20 Sep 2012 18:01:18 +0200
Subject: [PATCH] Increase /socket/timed_wait test max time

On slow architectures like armel, this test often fails with something like

  /socket/timed_wait:
GLib-GIO:ERROR:./gio/tests/socket.c:620:test_timed_wait: assertion failed
  (poll_duration < 110000): (154022 < 110000)
FAIL

Bump the maximum time.
---
 gio/tests/socket.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/tests/socket.c b/gio/tests/socket.c
index 0a61c9f..cd92e91 100644
--- a/gio/tests/socket.c
+++ b/gio/tests/socket.c
@@ -617,7 +617,7 @@ test_timed_wait (void)
   poll_duration = g_get_monotonic_time () - start_time;
 
   g_assert_cmpint (poll_duration, >=, 98000);
-  g_assert_cmpint (poll_duration, <, 110000);
+  g_assert_cmpint (poll_duration, <, 160000);
 
   g_socket_close (client, &error);
   g_assert_no_error (error);
-- 
1.7.10.4

